docs-chef-io/content/server/v14/config_rb_server_optional_settings.md

```diff
@@ -1508,6 +1508,16 @@ If new_size or heap_size is also specified directly in java_opts, it will be ign
 
 {{< /note >}}
 
+`elasticsearch['es_version']`
```

Comment:
This will only be supported until we finish upgrading internal es to 7, correct?